recipes/cuda-nvcc/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "cuda-nvcc" %}
+{% set version = "12.0.76" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvcc/{{ platform }}/cuda_nvcc-{{ platform }}-{{ version }}-archive.{{ extension }}
+
+  sha256: 160ca9f8828f53daa4b2066c5361aacb45fa2575885f70c223cda1d11df53d6f  # [linux64]
+  sha256: dc5bd7eac57b32a9645c0c23c6a1ba8a9533e846eeec31414e1aa4230afee2f8  # [ppc64le]
+  sha256: 85c8c643ea2cfa398e1b5ffcd597f5bb3b738526c9e3fb8e39dd909c55345f36  # [aarch64]
+  sha256: f67406f44263f0e4a110a8cf0200bff1d3d0e32917db8a7c9e417a87eb7ca372  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  missing_dso_whitelist:
+    - /lib64/libpthread.so.0
+    - /lib64/libdl.so.2
+    - /lib64/libm.so.6
+    - /lib64/libgcc_s.so.1
+    - /lib64/libc.so.6
+    - /lib64/ld-linux-x86-64.so.2
+    - /lib64/librt.so.1
+    - /lib64/libstdc++.so.6
+
+requirements:
```

Would it make sense to add the C & C++ compilers as `build` dependencies?

This would alleviate the need for the `missing_dso_whitelist` as well as letting the compiler runtime libraries be dropped from `host` and `run`

```suggestion
requirements:
  build:
    - {{ compiler("c") }}
    - {{ compiler("cxx") }}
```